docs/administration-guide/gaffer-deployment/gaffer-docker/gaffer-images.md

```diff
@@ -0,0 +1,115 @@
+# Gaffer Images
+
+As demonstrated in the [quickstart](../quickstart.md) it is very simple to start
+up a basic in memory gaffer graph using the available Open Container Initiative
+(OCI) images.
+
+For large scale graphs with persistent storage you will want to use a different
+storage backend to a Map Store; the recommended one being Accumulo. To do this a
+different deployment of containers are required. This guide will run through the
+containers needed for a basic Accumulo cluster and how to configure and create
+custom images of Gaffer.
+
+## Available Images
+
+Currently there are a few different images that can be used to run a Gaffer
+deployment. The main ones are outlined in the following table and are all
+available on [Docker Hub](https://hub.docker.com/u/gchq).
+
+| Image | Description |
+| ----- | ----------- |
+| `gchq/accumulo` | This image is a containerised deployment of [Apache Accumulo](https://accumulo.apache.org/). This was created as historically there has not been an available official image from the maintainers of Accumulo; however, there has since been an [offical image](https://github.com/apache/accumulo-docker) made available but it is not currently in use in Gaffer. |
```

Comment:
I think the official image is not very well supported. I also don't think it creates separate containers for the different parts of Accumulo (master, tserver, gc, etc.) whereas the Gaffer one does.

Reply:
I'll add a comment to say its not massively supported but I do think it creates separate containers like the Gaffer one where you pass an argument to run the Accumulo service you want the container to act as see: https://github.com/apache/accumulo-docker/blob/main/README.md?plain=1#L95-L104